### ISSUE TITLE
Maximum pokemon level too high

### ISSUE THREAD

**Issue by xZise** [farkam135/GoIV#1032]
When I'm scanning Pokemons and it is at the maximum level it'll shows "33-35" while I'm at level 31. My understanding is that the level can't be higher than `trainer level + 2` so I'm wondering if my understanding is wrong. Also modifying the level shows no movement for the levels 33 and above.

![screenshot_20190119-203549_pokmon go](https://user-images.githubusercontent.com/417478/51431697-c58cf980-1c2c-11e9-862f-7dcb216e54f3.jpg)

Interestingly on my friends phone wiht a level of 32, the maximum level Pokemon shows a level between 34 and 35.

**Comment by TotalKnob** [comment]
False, a weather booster Pokemon can have a level of the trainer level +2 +5. The maximum level of Pokemon found in the wild is 30, though weather boosted go up to level 35.
